service/src/main/java/org/ehrbase/aql/sql/binding/CompositionAttribute.java

```diff
@@ -40,7 +40,7 @@ public CompositionAttribute(CompositionAttributeQuery compositionAttributeQuery,
     public MultiFields toSql(I_VariableDefinition variableDefinition, String templateId, String identifier){
         MultiFields qualifiedAqlFields;
 
-        if (variableDefinition.getPath() != null && variableDefinition.getPath().startsWith("content")) {
+        if (variableDefinition.getPath() != null && (variableDefinition.getPath().startsWith("content") || variableDefinition.getPath().startsWith("context/other_context"))) {
```

Comment:
@darabosm can reuse the constant from JsonbEntryQuery ? https://github.com/ehrbase/ehrbase/pull/910/files#diff-a02634db48b227821e68d13543c4bf771a494c6de7f75c0e3a56429208674eb6R64